master.go

```diff
@@ -58,6 +58,15 @@ var masterRpcClientMap = map[int]*rpc.Client {}
 const NODE_SERVER = 0
 const NODE_CLIENT = 1
 
+type PutArgs struct {
```

Comment:
Isn't these the duplicate of PutArgs and GetArgs defined in client.go? Not sure how it doesn't throw error.

Reply:
Move these structs to shared/shared_structs.go because they will be need to be accessed in both client and master.